static/src/stylesheets/module/commercial/_adverts.scss

```diff
@@ -0,0 +1,330 @@
+        display: flex;
+
+        > .adverts__header {
+            box-sizing: border-box;
+            width: 240px;
+        }
+
+        > .adverts__body {
+            flex: 1;
+        }
+
+        .adverts__ctas {
+            margin-left: (-1 * $gs-gutter / 2);
+            margin-right: (-1 * $gs-gutter / 2);
+        }
+    }
+
+    > .adverts__header {
+        padding-bottom: $gs-baseline;
+        padding-top: $gs-baseline;
+
+        .inline-logo-guardian {
+            @include mq(leftCol) {
+                display: none;
+            }
+        }
+    }
+
+    > .adverts__body {
+        @include mq(mobileLandscape) {
+            box-sizing: border-box;
+            max-width: gs-span(8) + (2 * $gs-gutter);
+        }
+        @include mq(tablet) {
+            max-width: gs-span(9) + (2 * $gs-gutter);
+        }
+        @include mq(desktop) {
+            max-width: gs-span(12) + (2 * $gs-gutter);
+        }
+    }
+
+    .adverts__logo {
+        display: flex;
+        flex-direction: column;
+
+        > * + * {
+            margin-top: $gs-baseline / 2;
+        }
+
+        svg {
+            display: block;
+            fill: #ffffff;
+        }
+    }
+
+    .advert {
+        position: relative;
+    }
+
+    * + .advert {
+        border-left: 1px solid colour(neutral-4);
+    }
+}
+
+.adverts--legacy-single {
+    .advert {
+        flex-basis: calc(75% - #{$gs-gutter});
+        flex-grow: 0;
+    }
+
+    .advert__image-container {
+        @include mq(tablet) {
+            position: absolute;
+            left: $gs-gutter / 2;
+            top: 0;
+            width: $gs-gutter*11;
+        }
+    }
+
+    .adverts__row > .button {
+        position: absolute;
+        top: $gs-baseline;
+        right: $gs-gutter / 2;
+    }
+
+    .advert__title,
+    .advert__meta,
+    .advert__standfirst {
+        margin-left: $gs-gutter * 11;
+        padding-left: $gs-gutter / 2;
+        padding-right: $gs-gutter / 2;
+    }
+}
+
+.adverts__header {
+    display: flex;
+    flex-direction: column;
+
+    @include mq(tablet) {
+        padding: 0 $gs-gutter;
+    }
+}
+
+.adverts__ctas,
+.adverts__stamp {
+    margin-top: auto;
+}
+
+.adverts__ctas {
+    display: flex;
+    flex-direction: column;
+
+    > * + * {
+        margin-top: $gs-baseline / 2;
+    }
+}
+
+.adverts__ctas + .adverts__stamp {
+    margin-top: 0;
+}
+
+.adverts__body {
+    position: relative;
+    padding-bottom: $gs-baseline;
+}
+
+.adverts__row {
+    @include mq(mobile, tablet) {
+        > * + * {
+            margin-top: $gs-baseline;
+        }
+    }
+
+    @include mq(tablet) {
+        display: flex;
+        padding: 0 $gs-gutter / 2;
+
+        > * {
+            margin: 0 $gs-gutter / 2;
+            flex: 1;
+        }
+
+        .adverts--legacy & > * {
+            margin-left: 0;
+            margin-right: 0;
+            padding: 0 $gs-gutter / 2;
+        }
+    }
+
+}
+
+.adverts__row--1x1x1x1 {
+    > * {
```

Comment:
I am not sure about * selectors. As far as I know browsers are matching selectors from right to left. So in theory browser is going to match * (everything on the page) and then > so it will select only descendants of `.adverts__row--1x1x1x1` from that massive list. So unless something changed lately this is against the CSS performance recommendations.